packages/product-react/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@wework/ray-core-react';
```

Comment:
Questionable! Not sure if we need this, but if it borks without it, let's figure it out.

Reply:
What does this do 🤔

Reply:
i believe it just says that the `@wework/ray-core-react` module has any type

Reply:
but this should be removed as the most current ray core react has typings

Reply:
Does it? I'll try bumping the version